django-stubs/core/signing.pyi

```diff
@@ -20,31 +25,35 @@ class JSONSerializer:
 def dumps(
     obj: Any,
     key: bytes | str | None = ...,
-    salt: str = ...,
+    salt: bytes | str = ...,
     serializer: type[Serializer] = ...,
     compress: bool = ...,
 ) -> str: ...
 def loads(
     s: str,
     key: bytes | str | None = ...,
-    salt: str = ...,
+    salt: bytes | str = ...,
     serializer: type[Serializer] = ...,
     max_age: int | timedelta | None = ...,
+    fallback_keys: list[str | bytes] | None = ...,
 ) -> Any: ...
 
 class Signer:
-    key: str
+    key: bytes | str
+    fallback_keys: list[bytes | str]
     sep: str
-    salt: str
+    salt: bytes | str
     algorithm: str
     def __init__(
         self,
+        *args: Unpack[tuple[()]],
```

Comment:
This to make stubtest happy.

Two other solutions:
- Add the `__init__` in the ignorelist
- Add an overload, one without `args` (and a empty stararg as it is currently on django's `main`), one with `*args: Any` and `typing_extensions.deprecated` applied to this overload

Reply:
Let's just use `Any`